Tests/src/artofillusion/procedural/ModuleTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
Please do add a proper license header when creating a new file. It helps with clarity, and I've heard that in some jurisdictions, lack of a license is interpreted in such a way that @peastman or I could (theoretically) face legal difficulties if we added the file to the project.